setup.py

```diff
@@ -82,7 +82,7 @@
 ]
 
 extras_require = setuptools_args['extras_require'] = {
-    'test': ['ipykernel', 'ipython', 'pytest'],
+    'test': ['ipykernel', 'ipython', 'mock', 'pytest', 'pytest-warnings'],
```

Comment:
I believe that 'pytest-warnings' is deprecated and the functionality is included in pytest >= 3.1. Is there a particular reason to include it here?